newbank/server/NewBank.java

```diff
@@ -1,55 +1,77 @@
 package newbank.server;
 
 import java.util.HashMap;
+import java.util.ArrayList;

```

Comment:
Import statements should be in alphabetical order (something I missed too when I made my changes)